plugins/TagFix_Vatin.py

```diff
@@ -58,11 +60,15 @@ def it_vatin(self, vatin):
     def node(self, data, tags):
         if "ref:vatin" in tags:
             if tags["ref:vatin"].startswith("IT"):
-                if self.it_vatin(tags["ref:vatin"][2:]) == False:
-                    return {"class": 32601, "subclass": 1 }
+                if self.it_vatin(tags["ref:vatin"][2:]) is False:
+                    return {"class": 99998, "subclass": 1, "text": {"en": u"Invalid 'Partita IVA'"}}
```

Comment:
Please use english text. Let's translators translate.

Reply:
fixed, of course is better to only on text for all future countries